extensions/azcli/src/az.ts

```diff
@@ -447,7 +447,7 @@ export async function checkAndInstallAz(userRequested: boolean = false): Promise
 	try {
 		return await findAzAndArc(); // find currently installed Az
 	} catch (err) {
-		if (err === AzureCLIArcExtError) {
+		if (err.toString() === 'Error: ' + loc.arcdataExtensionNotInstalled) {
```

Comment:
It would be better to have it throw a specific error type for this (like NoAzureCLIError) and then check for that with `err instanceof <type name here>`